app/index.js

```diff
@@ -405,7 +428,10 @@ function showThankYou() {
     if (BodyPresenceSensor) {
         feedbackData['bodyPresence'] = bodyPresence.present;
     }
-    console.log(feedbackData['responseSpeed']);
+
+    feedbackData['restingHR'] = (user.restingHeartRate || 999);
```

Comment:
I don't like this. Can't you just not feedback this data?

Reply:
As in delete these lines, and let the backend not input the data

Reply:
@FedericoTartarini I'm ready to merge this branch. Just need to get feedback from you as to whether these lines could be deleted?